sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_models.py

```diff
@@ -299,8 +317,7 @@ class TranslationStatus(
     :ivar int total_characters_charged: Total characters charged across all documents within the translation operation.
     """
 
-    def __init__(self, **kwargs):
-        # type: (**Any) -> None
+    def __init__(self, **kwargs: Any) -> None:
```

Comment:
this model and everything below is an output model only so don't feel compelled to add the kwonly args to the init